linkcheck/strformat.py

```diff
@@ -194,12 +195,20 @@ def strsize(b, grouping=True):
     if b < 1024 * 1024:
         return "%sKB" % locale.format_string("%.2f", (float(b) / 1024), grouping)
     if b < 1024 * 1024 * 10:
-        return "%sMB" % locale.format_string("%.2f", (float(b) / (1024*1024)), grouping)
+        return "%sMB" % locale.format_string(
+            "%.2f", (float(b) / (1024 * 1024)), grouping
+        )
     if b < 1024 * 1024 * 1024:
-        return "%sMB" % locale.format_string("%.1f", (float(b) / (1024*1024)), grouping)
+        return "%sMB" % locale.format_string(
+            "%.1f", (float(b) / (1024 * 1024)), grouping
+        )
     if b < 1024 * 1024 * 1024 * 10:
-        return "%sGB" % locale.format_string("%.2f", (float(b) / (1024*1024*1024)), grouping)
-    return "%sGB" % locale.format_string("%.1f", (float(b) / (1024*1024*1024)), grouping)
+        return "%sGB" % locale.format_string(
+            "%.2f", (float(b) / (1024 * 1024 * 1024)), grouping
+        )
+    return "%sGB" % locale.format_string(
```

Comment:
oh god, we wrote our own thing like that as well? and we stop at GBs? maybe we should make a not to replace that with something cleaner, like:

https://github.com/borgbackup/borg/blob/0a0eaf05e6dfd18428d7a2ed85f21b8f805a7468/src/borg/helpers/parseformat.py#L273

see also 

https://stackoverflow.com/a/1094933

Reply:
Opened  #433.

Reply:
great, thanks.